guillotina/db/storages/pg.py

```diff
@@ -782,21 +782,21 @@ def objects_table_name(self):
 
     @restart_conn_on_exception
     async def open(self):
-        conn = await self.pool.acquire(timeout=self._conn_acquire_timeout)
+        conn = await self.pool.acquire()
         return conn
 
     async def close(self, con):
         try:
-            await shield(asyncio.wait_for(self.pool.release(con, timeout=1), 1))
+            await shield(self.pool.release(con, timeout=1))
```

Comment:
I wonder why `wait_for` was used along with the release timeout... What does this cause that is a problem?

Reply:
I'm not sure if this is a problem or not, I can put it back